### ISSUE TITLE
reportUnusedVariable is enabled by default and can not be disabled

### ISSUE THREAD

**Issue by last-partizan** [fannheyward/coc-pyright#85]
This plugin marks unused variables, hovewer, reportUnusedVariable is disabled by default.

Like in this example:
```
def test(param):
    pass
```

![image](https://user-images.githubusercontent.com/301015/90339553-24c61200-dffa-11ea-8800-7a7d3a6607fe.png)

i tried to disable it by adding this to CocConfig:

```
  "python.analysis.diagnosticSeverityOverrides": {
    "reportUnusedVariable": "none"
  },
```

But it does not works.

Does this looks like plugin problem, or maybe i should create issue in pyright repo?

**Comment by fannheyward** [comment]
Test with Pyright + VSCode, same issue.

**Comment by last-partizan** [comment]
@fannheyward so, this looks like pyright-langserver issue, and must be reported there?

**Comment by fannheyward** [comment]
Yes, looks like Pyright doesn't resolve the configuration correctly.

**Comment by yaegassy** [comment]
@fannheyward 

You asked this question on the pyright gitter, and I'll post a link to this issue as well.

https://gitter.im/microsoft-pyright/community?at=5f3a8e16b7818b3998021c74

The diagnostic part of this issue **does not** seem to be controlled by `python.analysis.diagnosticSeverityOverrides`.

**Comment by yaegassy** [comment]
@fannheyward @last-partizan 

As for the current situation of `coc-pyright` and `pyright` itself, the only way to disable the diagnostic "hint messages" is to set the `diagnostic.level` to `information` or higher (`warning` or `error`).

You might want to use `:CocLocalConfig` to create and configure a "project-level" `.vim/coc-settings.json`.

----

**.vim/coc-settings.json**

```jsonc
{
  "diagnostic.level": "information"
}
```

**Comment by last-partizan** [comment]
@yaegassy thank you.

For now i configured another workaround - i was using coc-diagnostic-next to jump between errors, but i switched to coc-diagnostic-next-error, so i can jump only to errors and able to see warnings also.
